### PR TITLE
implemented the Go SDK MVP with support for customer cards related types

### PR DESCRIPTION
**Pull request by presmihaylov**
* Added all the types one needs to implement a customer card API
* Added an example http server, implementing a customer card endpoint
* Updated README with initial directions & clarifications

### CODE REVIEW

File: customercards/customercards.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package customercards
+
+import gosdk "github.com/team-plain/go-sdk"
+
+type Request struct {
```
Comment (presmihaylov) [inline customercards/customercards.go:5]: double-check this from zod schema within codebase

File: types.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
+package gosdk
```
Comment (presmihaylov) [inline types.go:1]: cross check this with types in internal codebase as the example repo I used might be outdated

File: README.md [hunk 3]
```diff
@@ -1,2 +1,8 @@
 # go-sdk
-This is the Go SDK for Plain's Core GraphQL API
+This is the Go SDK for Plain's Core GraphQL API.
+
+It is not complete yet with all the types we support in our API, but it provides all the types one needs in order to implement a customer cards API ([more info](https://www.plain.com/docs/customer-cards)).
```
Comment (mattvagni) [inline README.md]:
```suggestion
This repository will, in future, be the home for a Go SDK for Plain's GraphQL API. 

Currently this only provides all the types one needs in order to implement a customer cards API ([more info](https://www.plain.com/docs/customer-cards)).
```

Comment (mattvagni) [reply README.md]: I think this is slightly more 'honest' 😉

**Review by mattvagni — APPROVED** [review]
lgtm!

**Pull request merged**